### ISSUE TITLE
ovmm delete_user does not work

### ISSUE THREAD

**Issue by solstice** [tobiasraabe/otree_virtual_machine_manager#121]
* oTree Virtual Machine Manager version: latest
* Python version: 3.5 / 3.6
* Operating System: oTree Virtual Machine image, but also fresh osboxes.org Ubuntu image

### Description

I wanted to add, and then delete a user. Adding works. When I try to delete a user, there are some errors, and the user does not get removed from the PG table and from the system. The file user_config/davidjk.txt is deleted after running the command, however, subsequent runs of the delete_user command still fail with reference to the pkill exit code.

Thanks for the great work on ovmm, it is really very helpful.

### What I Did

```
sudo ovmm delete_user

------------------- Process: Delete User -------------------
Do you want to see a list of users? [Y/n]: n
Which user do you want to delete? [louis]: davidjk
Do you want a database backup? [Y/n]: n
ERROR: User could not be deleted.
Command line: ['/usr/bin/sudo', 'pkill', '-u', 'davidjk']
Exit code: 0
SUCCESS: nginx configuration works.
SUCCESS: nginx configuration works.
SUCCESS: Removed user from nginx configuration.
SUCCESS: smb.conf works.
SUCCESS: Created backup of smb.conf.
SUCCESS: Removed user from smb.conf.
ERROR: User configuration text file could not be deleted.
Command line: ['/usr/bin/sudo', 'rm', '/home/otreeadmin/ovmm_sources/user_configs/davidjk.txt']
Exit code: 0
ERROR: An exception was raised during the deletion process.
Please, fix the problem manually. After that, run the command
again, so that the user will also be deleted from database.

WARNING: The deletion process is completed. If an error
occurred, check it manually.

----------------------- Process: End -----------------------
```

**Comment by tobiasraabe** [comment]
Hi solstice,

Thanks for your feedback. I am glad to hear that this piece provides some benefits for you.

Regarding the issue, which version are you using? Have you installed it via pip or directly from the repo?
 We have made some adjustments after the latest release of version 0.2.2 and at least the second error is probably solved. I opened a PR (#122) with some instructions. Have a look at this.

*Side note on what is going on*:
I think that everything will work when the two exit codes are ignored. Exit codes of 0 normally denote a successfully terminated process. The user and her home folder is not removed when her processes cannot be killed. Second, ``davidjk.txt`` was deleted in a previous run and now the command returns error since it has successfully terminated, but did not delete anything.

Best
